knpu/ep3-webpack/babel.md

```diff
@@ -23,8 +23,10 @@ Copy the `yarn add` line, though we already have `webpack` installed.
 
 Find your terminal and run it with `--dev` on the end:
 
+>Here we have problem! We need to use babel-loader@7!
```

Comment:
So what's the deal with that kind of changes? Do we want to add a `TIP` block about the required changes in order to work with Webpack4?

Reply:
I think Vladimir just wanted to left us a bit more context for review and left his comments in scripts, but before we're able to merge it - we need to remove them completely and left only real changes like `yarn add babel-loader@7` below or convert these comments into real notes using our `***TIP` syntax that probably even better because we'll give our more context, like WHY they should use `babel-loader@7` here etc.

Reply:
Still need some more info here!